tests/Core/Components/Divider/FluentDividerTests.razor

```diff
@@ -0,0 +1,76 @@
+﻿@using Microsoft.FluentUI.AspNetCore.Components.Tests.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Utilities
+@using Xunit;
+@inherits TestContext
+
+@code
+{
+    public FluentDividerTests()
+    {
+        JSInterop.Mode = JSRuntimeMode.Loose;
+        Services.AddFluentUIComponents();
+    }
+
+    [Fact]
+    public void FluentDivider_Default()
+    {
+        // Arrange && Act
+        var cut = Render(@<FluentDivider>Hello World</FluentDivider>);
+
+        // Assert
+        cut.Verify();
+    }
+
+    [Theory]
+    [InlineData(DividerAlignContent.Center, "center")]
+    [InlineData(DividerAlignContent.Start, "start")]
+    [InlineData(DividerAlignContent.End, "end")]
```

Comment:
You need to also test:
 - `[InlineData(null, "center")]`
 - `[InlineData((DividerAlignContent)999, "center")]`